openlibrary/views/loanstats.py

```diff
@@ -32,27 +42,40 @@ def reading_log_summary():
     reading_log_summary, 'stats.readling_log_summary', timeout=dateutil.HOUR_SECS
 )
 
-def cached_get_most_logged_books(shelf_id=None, since_days=1, limit=20):
-    return cache.memcache_memoize(
-        get_most_logged_books, 'stats.trending', timeout=dateutil.HOUR_SECS
-    )(shelf_id=shelf_id, since_days=since_days, limit=limit)
-
-def get_most_logged_books(shelf_id=None, since_days=1, limit=20):
-    """
-    shelf_id: Bookshelves.PRESET_BOOKSHELVES['Want to Read'|'Already Read'|'Currently Reading']
-    since: DATE_ONE_YEAR_AGO, DATE_ONE_MONTH_AGO, DATE_ONE_WEEK_AGO, DATE_ONE_DAY_AGO
-    """
-    # enable to work w/ cached
-    if 'env' not in web.ctx:
-        delegate.fakeload()
+@public
+def get_trending_books(since_days=1, limit=20, page=1, books_only=False):
```

Comment:
```suggestion
def get_trending_books(since_days=1, limit=18, page=1, books_only=False):
```
Setting default limit to `18` here prevents duplicate books from being fetched asynchronously.